connectors/bigtable/flink-connector-gcp-bigtable/src/main/java/com/google/flink/connector/gcp/bigtable/serializers/GenericRecordToRowMutationSerializer.java

```diff
@@ -97,6 +120,21 @@ private RowMutationEntry serializeWithColumnFamily(
         return entry;
     }
 
+    @Internal
```

Comment:
do we need these internal methods? can't we just access the private field on the class?

Reply:
AFAIK, Java doesn't allow getting private fields outside the class. We could use [reflection](https://www.oracle.com/technical-resources/articles/java/javareflection.html) but I think it's overkill, hence the getters

Reply:
why are they private? if we do "public final" they can be visbible outside the class but readonly once set

Reply:
Isn't that an anti pattern for fields generated from a Builder?

My understanding is that, unless you truly want to have those fields accessible publicly outside the class (which is not the case, it's better to have getters to ensure encapsulation and immutability

https://softwareengineering.stackexchange.com/a/190959

This is what the BQ connector did:
https://github.com/GoogleCloudDataproc/flink-bigquery-connector/blob/main/flink-1.17-connector-bigquery/flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/sink/writer/BigQueryBufferedWriter.java#L391

I honestly DON'T like getters for small final cases (like this), so if we both prefer public, I am OK with it, I was just trying to follow the best practices I had

Reply:
Removed the getters